### ISSUE TITLE
rounding problems with Choices (Model and MultipleRuns)

### ISSUE THREAD

**Issue by pedro-andrade-inpe** [TerraME/calibration#180]
When both `Model` and `MultipleRuns` have a parameter `Choice` with `step`, there might be rounding errors that does not allow a model to be instantiated. It seems that the error is in `MultipleRuns`.

```lua
Model{
    value = Choice{min = 0, max = 1.0, step = 0.1}
    -- ...
}

MultipleRuns{
    -- ...
    parameters = {
        value = Choice{min = 0.1, max = 0.9, step = 0.1}
    }
    -- ...
}
```

**Comment by agojunior** [comment]
executing this code returns: Error: Invalid value for argument 'parameters.value' (0.5).

However 0.5 IS a valid argument.

It appears the error happens in line 292 of multiple runs which is:
 local m = data.model(mVariables) --testing the model with it's current parameter values.

However there is no customError in multipleRuns.lua or utils.lua code that return this sentence "invalid value for argument..."

the part of the code that verifies if a value with step is within a Model acceptable values is this one: 
if values.step ~= nil then
		if Param.step == nil then
			customError("Argument '"..idx..".step' is mandatory.")
		elseif Param.step % values.step ~= 0 then
			customError("Parameter "..idx.." step is out of the model range.")
		end

		if values.min ~= nil then
			if (Param.min - values.min) % values.step ~= 0 then
				customError("Parameter "..idx.." min is out of the model range.")
			end
		end
	end

however this is not the part of the code where this error is coming from.

In other words... i have no idea which part of the code is making this error.

**Comment by pedro-andrade-inpe** [comment]
It might be an error in TerraME. I'll check that.

**Comment by agojunior** [comment]
[yay.txt](https://github.com/TerraME/calibration/files/1350055/yay.txt)
